frontend/src/app/components/resizer/wp-resizer.component.ts

```diff
@@ -133,13 +145,35 @@ export class WpResizerDirective extends UntilDestroyedMixin implements OnInit, A
     // Send a event that we resized this element
     const event = new Event(this.resizeEvent);
     window.dispatchEvent(event);
+
+    if (this.resizer.classList.contains('-error')) {
```

Comment:
As you use this code block three times only with slight differences, it may make sense to extract it into a separate method:
```
private toggleErrorClass(val:boolean){
  if (..contains...) {
    jQuery(this.resizer).toggleClass('-error', val)
  }
}
```

Reply:
Thanks for your feedback @HDinger , it made me think. I understand what you mean, and I agree with extracting logic to reusable methods.

I think this case is slightly different from a toggle scenario though. In order to avoid unnecessary DOM manipulations (resizeMove is triggered a lot of times), what I wanted here is to add '-error' class only if it isn't present (lines 149 & 173) and remove the '-error' class only if it is present (line 167). So it is a kind of a toggleIf, something like:

private toggleErrorClass(val:boolean){
  if ((val && !this.resizer.classList.contains('-error')) || 
      (!val  && this.resizer.classList.contains('-error'))) {
    jQuery(this.resizer).toggleClass('-error', val)
  }
}

To me, it looks much clearer the DOM API way, it is standard, more semantic and I don't see too much logic to abstract away in this case. I also try to avoid using third party libraries for DOM manipulations when possible.

Mixing both perspectives, I came up with this:

 ` private manageErrorClass(shouldBePresent:boolean) {
    if (shouldBePresent && !this.resizer.classList.contains('-error')) {
       this.resizer.classList.add('-error');
    }

    if (!shouldBePresent && this.resizer.classList.contains('-error')) {
      this.resizer.classList.remove('-error');
    }
  }`

Would it be ok for you?

Looking forward to hearing your thoughts on this.

Reply:
I like your suggestion. 👍  In general, of course, we should try to make as few calls as possible. My suggestion was also not meant to simply adopt it, but to clarify what I mean. For me, it's just important that similar code parts don't duplicate themselves too often. And this is fulfilled with your suggested method. :)

Reply:
Cool, thanks 👍